demos/ready.js

```diff
@@ -37,7 +37,8 @@ window.demoReady = (function(root) {
       return true;
     }
     ensureDetectionDom();
-    isReadyCached = Boolean(window.mdc) && getComputedStyle(testDom).position === 'relative';
+    isReadyCached = getComputedStyle(testDom).position === 'relative' &&
+      (Boolean(window.mdc) || !root.querySelector('script[src$="material-components-web.js"]'));
```

Comment:
It might be more resilient if we use a `[src*="value"]` selector instead of `[src$="value"]`.

That way, if we ever need to add a query string to the end of the URL for some reason (e.g., cache busting for statically-hosted PR screenshot tests), this won't suddenly break (which might be hard to debug, especially for anyone who joins the team after this code was added).